x/vesting/module.go

```diff
@@ -135,11 +134,6 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	}
 }
 
-// InitGenesis performs a no-op.
-func (am AppModule) InitGenesis(_ sdk.Context, _ codec.JSONCodec, _ json.RawMessage) []abci.ValidatorUpdate {
-	return []abci.ValidatorUpdate{}
-}
```

Comment:
unrelated to these changes but why does the vesting module not have the same interface assertions as the other modules? e.g. `_ appmodule.HasABCIGenesis = AppModule{}`